### ISSUE TITLE
Update to Elm 0.15.1 (from 0.12)

### ISSUE THREAD

**Issue by dusky3** [elm-lang/debug.elm-lang.org#9]
I was playing with the online debugger and even the following simple example that works at _http://elm-lang.org/try_ throws a bunch of errors:

```
import Graphics.Element exposing (show)
main = show (toString 42)


Parse error at (line 1, column 25):
unexpected "e"
expecting "{-", " ", newline, reserved word 'as' or "("
```

When I remove _exposing_ it throws this:

```
import Graphics.Element (show)
main = show (toString 42)


Error on line 2, column 14 to 22:
Could not find variable 'toString'
```

because as I can see in the hint it tries to import _toString_ from JSON.toString. If I click on the hint it points to _http://library.elm-lang.org/catalog/evancz-Elm/0.12/Json#toString_ and I get _Server not found_ message.

It does't work even when I try this:

```
import Basics
import Graphics.Element (show)
main = show (Basics.toString 42)


Error on line 3, column 14 to 29:
Could not find variable 'Basics.toString'.
```

I believe that _http://elm-lang.org/try_ and _http://debug.elm-lang.org/try_ should use the same (latest) version of Elm otherwise the debugger is not usable at all.

BTW: the Try it! buttons next to the examples at _http://debug.elm-lang.org_ are are not clickable, but if you click on the Debug it takes you to _http://debug.elm-lang.org/try_ where you can see the examples (written in Elm 0.12).

**Comment by jvoigtlaender** [comment]
According to the README of this repository, it is not really maintained anymore.

Maybe for the sake of keeping the http://debug.elm-lang.org/ site alive, it would make sense to update stuff nevertheless. But it could be a lot of work.

**Comment by dusky3** [comment]
I can read that it is not maintained anymore in the README, but when you come to http://debug.elm-lang.org/ it doesn't say anything there. There are lot of tutorials out there which suggest using the online version of the debugger, so it would be nice to include this information there as well.

**Comment by jvoigtlaender** [comment]
Right. Up to @evancz to decide what form of warning to put on the http://debug.elm-lang.org/ page that while the examples there do work, newer code that is valid current Elm might not.

**Comment by iqualfragile** [comment]
actually update to 0.17.0

**Comment by doppioslash** [comment]
@iqualfragile I doubt it's that easy. Would need to have elm-reactor's time travelling debugging back, to begin with.

**Comment by iqualfragile** [comment]
there is http://package.elm-lang.org/packages/jinjor/elm-time-travel/1.0.6/
maybe that can be integrated?
